lib/booktype/apps/core/static/core/js/booki.js

```diff
@@ -760,15 +760,16 @@
       */
       notification: function (title, message, settings) {
         var opts = win.booktype.editor.data.settings.config.notifications.settings;
+        var icon = 'icon-bullhorn';
 
         jquery.notify({
-            icon: settings.icon || 'icon-bullhorn',
```

Comment:
We have a little problem here, we need this to be configurable and not just be tight to `icon-bullhorn`, for instance: sometimes you would need another icon or even pass empty string to not show icon. Please put back the settings.icon || default back ;)